delayed inbox configuration (#1152)
OffchainLabs/arbitrum-docs: arbitrum-docs/launch-orbit-chain/how-tos/orbit-chain-finality.md

# Orbit Chain Confirmation and Finality

> ## Child Chain Transactions

Reviewer: ```suggestion
## Child chain transactions
```
Titles should be in sentence-case ([Style conventions](https://docs.arbitrum.io/for-devs/contribute#style-conventions))

Reviewer: (Apply on all other content labels 🙏 )

> Generally, transactions executed through the sequencer on Orbit chains [achieve finality](/tx-lifecycle) equivalent to their parent chain once the relevant transaction data has been [posted in a batch](/sequencer). This means that transactions on Orbit chains are considered final in minutes.

Reviewer: ```suggestion
Generally, transactions executed through the sequencer on Orbit chains [achieve finality](/tx-lifecycle.mdx) equivalent to their parent chain once the relevant transaction data has been [posted in a batch](/sequencer.dx). This means that transactions on Orbit chains are considered final in minutes.
```
Links should point to the file with the extension, this way Docusaurus can check if the link is broken or not.

Also, check this FAQ, in case you find it to be a better reference for the finality link: https://arbitrum-docs-git-orbit-finality-explanation-offchain-labs.vercel.app/for-devs/troubleshooting-building#how-many-block-numbers-must-we-wait-for-in-arbitrum-before-we-can-confidently-state-that-the-transaction-has-reached-finality

Reviewer: (Please, also change the other links of the docs 🙏 )

## Parent Chain → Child Chain Transactions

> Messages being sent through the delayed inbox of a parent chain as [retryable tickets](/arbos/l1-to-l2-messaging#retryable-tickets), including deposits through token bridges, are released by a [well-behaved Sequencer](/inside-arbitrum-nitro/#if-the-sequencer-is-well-behaved) once they have good confidence of finality on a parent chain. On an L2 settling to Ethereum, this means that a sequencer will release delayed messages to the inbox after 40 blocks. Following this, the transaction will need to complete another finality period for the Ethereum transaction that promoted them to achieve finality.

Reviewer: ```suggestion
Messages being sent through the delayed inbox of a parent chain as [retryable tickets](/arbos/l1-to-l2-messaging.mdx#retryable-tickets), including deposits through token bridges, are released by the [sequencer](/inside-arbitrum-nitro/inside-arbitrum-nitro.mdx#if-the-sequencer-is-well-behaved) once it has good confidence of finality on the parent chain. For example, on an L2 chain settling to Ethereum, the sequencer will release delayed messages to the inbox after 40 blocks. Following this, the transaction will need to complete another finality period for the Ethereum transaction that promoted it to achieve finality.
```
Rewording it a bit to make it a bit more concise, but feel free to take and dismiss any of those changes.
(This is a non-blocking comment)

> Messages being sent through the delayed inbox of a parent chain as [retryable tickets](/arbos/l1-to-l2-messaging#retryable-tickets), including deposits through token bridges, are released by a [well-behaved Sequencer](/inside-arbitrum-nitro/#if-the-sequencer-is-well-behaved) once they have good confidence of finality on a parent chain. On an L2 settling to Ethereum, this means that a sequencer will release delayed messages to the inbox after 40 blocks. Following this, the transaction will need to complete another finality period for the Ethereum transaction that promoted them to achieve finality.

Reviewer 2: ```suggestion
Messages being sent through the delayed inbox of a parent chain as [retryable tickets](/arbos/l1-to-l2-messaging.mdx#retryable-tickets), including deposits through token bridges, are released by the [sequencer](/inside-arbitrum-nitro/inside-arbitrum-nitro.mdx#if-the-sequencer-is-well-behaved) once it has reasonable confidence of finality on the parent chain. For example, on an L2 chain settling to Ethereum, the sequencer will release delayed messages to the inbox after 40 blocks. Following this, the transaction must complete another finality period for the Ethereum transaction that promoted it to achieve finality.
```

> Orbit L3s may configure the finality of transactions executed through the delayed inbox to depend on different layers of finality. By default, Orbit chains will rely on the number of L1 block confirmations, effectively finalizing an L3 deposit as soon as L1 finalizes the batch posted by Arbitrum One or when a DACert is posted by Arbitrum Nova. This would be on the order of tens of minutes.

Reviewer 2: ```suggestion
Orbit L3s may configure the finality of transactions executed through the delayed inbox to depend on different layers of finality. By default, Orbit chains will rely on the number of L1 block confirmations, effectively finalizing an L3 deposit as soon as L1 finalizes the batch posted by <a data-quicklook-from="arbitrum-one">Arbitrum One</a> or when a DACert is posted by <a data-quicklook-from="arbitrum-nova">Arbitrum Nova</a>. This would be on the order of tens of minutes.
```

However, in the instance of an L3 settling to Arbitrum One or Nova an L3 may also choose to rely only on L2 finality by configuring their sequencer as follows:

```
--node.delayed-sequencer.use-merge-finality=false
```

Additionally, the delay in L3 finalization can be decreased to achieve extremely fast (<1 min) deposits by configuring the sequencer to wait for fewer L2 block confirmations:

```
--node.delayed-sequencer.finalize-distance=1
```

Note, however, that if you choose to enable fast bridging, a re-org of un-finalized blocks on the L3 may occur if Arbitrum One/Nova (or the settlement chain of choice) experiences a re-org.

## Child Chain → Parent Chain Transactions

> Normally, [outgoing transactions](/arbos/l2-to-l1-messaging) must wait until the assertion that includes their L2 message to confirm (~1 week) before a client can execute the message on L1. However, in the near future [Anytrust](/inside-anytrust) chains will have the ability to leverage their DAC to enable fast confirmations of withdrawals through the native token bridge. By immediately confirming assertions that have been signed by the DAC, finality can be reduced to ~15 minutes.

Reviewer: ```suggestion
Normally, [outgoing transactions](/arbos/l2-to-l1-messaging) must wait until the assertion that includes their L2 message is confirmed (~1 week) before a client can execute the message on L1. However, in the near future [Anytrust](/inside-anytrust) chains will have the ability to leverage their DAC to enable fast confirmations of withdrawals through the native token bridge. By immediately confirming assertions that have been signed by the DAC, finality can be reduced to ~15 minutes.
```
(I think?)

> Normally, [outgoing transactions](/arbos/l2-to-l1-messaging) must wait until the assertion that includes their L2 message to confirm (~1 week) before a client can execute the message on L1. However, in the near future [Anytrust](/inside-anytrust) chains will have the ability to leverage their DAC to enable fast confirmations of withdrawals through the native token bridge. By immediately confirming assertions that have been signed by the DAC, finality can be reduced to ~15 minutes.

Reviewer 2: ```suggestion
Normally, [outgoing transactions](/arbos/l2-to-l1-messaging) must wait until the assertion that includes their L2 message is confirmed (~1 week) before a client can execute the message on L1. However, in the near future, [AnyTrust](/inside-anytrust) chains will be able to leverage their DAC to enable fast confirmations of withdrawals through the native token bridge. By immediately confirming assertions that have been signed by the DAC, finality can be reduced to under fifteen minutes.
```
